Source/WebCore/editing/MarkupAccumulator.cpp

```diff
@@ -664,7 +665,7 @@ LocalFrame* MarkupAccumulator::frameForAttributeReplacement(const Element& eleme
 Attribute MarkupAccumulator::replaceAttributeIfNecessary(const Element& element, const Attribute& attribute)
 {
     if (element.isHTMLContentAttribute(attribute)) {
-        auto frame = frameForAttributeReplacement(element);
+        RefPtr frame = frameForAttributeReplacement(element);
```

Comment:
Note line 324 also has `auto frame = frameForAttributeReplacement(element)`, which should also be updated to `RefPtr`.

Reply:
Can we address this comment either in this PR or in a follow up?

Reply:
I can update it here!

Reply:
Thank you!